.github/workflows/basic_cli_build_release.yml

```diff
@@ -140,3 +139,43 @@ jobs:
           path: |
             ${{ env.TAR_FILENAME }}
 
+  test-release-ubuntu:
+    needs: [create-brotli-archive]
+    runs-on: [ubuntu-20.04]
+    steps:
+
+      - name: Download the previously uploaded files
+        uses: actions/download-artifact@v3
+
+      - name: mv roc nightly and simplify name
+        run: mv $(ls -d artifact/* | grep "roc_nightly.*tar\.gz" | grep "linux") ./roc_nightly.tar.gz
+
+      - name: decompress the tar
+        run: tar -xzvf roc_nightly.tar.gz
+
+      - name: delete tar
+        run: rm roc_nightly.tar.gz
+
+      - name: rename nightly folder
+        run: mv roc_nightly* roc_nightly
+
+      - run: cd basic-cli-platform && ls | grep "tar" | xargs tar -xzf
+
+      - name: prep testing http-get.roc
+        run: |
+          mv roc_nightly basic-cli-platform/.
+          cd basic-cli-platform
+          mkdir examples
+          cd examples
+          curl -OL https://raw.githubusercontent.com/roc-lang/basic-cli/main/examples/http-get.roc
+          sed -i 's/pf:\ \"[^"]*/pf:\ \"\.\.\/main.roc/g' http-get.roc
+          cd ..
+          curl -OL https://raw.githubusercontent.com/roc-lang/basic-cli/main/ci/expect_scripts/http-get.exp
+
+      - run: sudo apt install -y expect
```

Comment:
`expect` allows us to easily test interactive programs.

Reply:
Huh, cool! I'd never heard of this before.

Reply:
chatGPT told me about it :)